### PR TITLE
Document the instant rate.

### LINKED ISSUE #234: Document irate
https://github.com/prometheus/prometheus/pull/1155

### PR DESCRIPTION
**Pull request by brian-brazil**
Hamming up the "angry" meaning just a tad.

Fixes #234

### CODE REVIEW

File: content/docs/querying/functions.md [hunk 1]
```diff
@@ -159,6 +159,29 @@ increase(http_requests_total{job="api-server"}[5m])
...
+
+The following example expression returns the per-second rate of HTTP requests
+looking up to 5 minutes back for the two most recent data points, per time
+series in the range vector:
+
+```
+irate(http_requests_total{job="api-server"}[5m])
+```
+
+`irate` should only be used when graphing volatile fast-moving counters.
```
Comment (fabxc) [inline content/docs/querying/functions.md]: comma?

File: content/docs/querying/functions.md [hunk 2]
```diff
@@ -159,6 +159,29 @@ increase(http_requests_total{job="api-server"}[5m])
...
+Use `rate` for alerts and slow-moving counters.
```
Comment (fabxc) [inline content/docs/querying/functions.md]: Even though those function specs should be short and precise, a brief reasoning would certainly be helpful here.

File: content/docs/querying/functions.md (hunk 1 above)
Comment (brian-brazil) [inline content/docs/querying/functions.md]: Done

File: content/docs/querying/functions.md (hunk 2 above)
Comment (brian-brazil) [inline content/docs/querying/functions.md]: Done

**Comment by juliusv** [comment]
:+1:

**Pull request merged**